src/containerapp/HISTORY.rst

```diff
@@ -2,8 +2,15 @@
 
 Release History
 ===============
-Upcoming
-+++++++
+0.3.22
+++++++
+* BREAKING CHANGE: 'az containerapp env certificate list' returns [] if certificate not found, instead of raising an error.
```

Comment:
Why do we need to do this breaking change?

Reply:
Previously, we were only querying for private-key certificates and showing whatever the API returned. Now with managed certificates added, we have to query two separate APIs to look for the matching certificates. So we can't show the API error directly. Considering it's a `list` command instead of `show` command, I think it makes more sense to return `[]` than raising an error.

Reply:
Got it, thanks~